README.md

```diff
@@ -4,31 +4,72 @@
+It contains standalone state-of-the-art components representative of a generic XR workflow,
+    as well as a runtime framework that integrates these components into an XR system.
+ILLIXR's runtime integration framework is modular, extensible, and [OpenXR][26]-compatible.
 
-We use the term _components_ and not _kernels_ or _computations_ because each component of ILLIXR is an entire application in itself, and consists of many kernels and computations. At the moment, ILLIXR contains the following state-of-the-art components, all of which can be found packaged together in the [v1-latest release](https://github.com/ILLIXR/ILLIXR/releases/tag/v1-latest) of ILLIXR.
+We use the term _components_ and not _kernels_ or _computations_ because
+    each component of ILLIXR is an entire application in itself,
+    and consists of many kernels and computations.
+At the moment, ILLIXR interfaces the following state-of-the-art components,
+    with periodic ILLIXR source snapshots hosted on our [releases page][25]:
 
-1. [Simultaneous Localization and Mapping](https://github.com/ILLIXR/open_vins)
-2. [Scene reconstruction](https://github.com/ILLIXR/ElasticFusion)
-3. [Eye tracking](https://github.com/ILLIXR/RITnet)
-4. [Ambisonic encoding](https://github.com/ILLIXR/audio_pipeline)
-5. [Ambisonic manipulation and binauralization](https://github.com/ILLIXR/audio_pipeline)
-6. [Lens distortion correction](https://github.com/ILLIXR/visual_postprocessing)
-7. [Chromatic aberration correction](https://github.com/ILLIXR/visual_postprocessing)
-8. [Time warp](https://github.com/ILLIXR/visual_postprocessing)
-9. [Computational holography for adaptive multi-focal displays](https://github.com/ILLIXR/HOTlab)
+1.  [Simultaneous Localization and Mapping][1] **
 
-We plan on adding more components (e.g., graphics and multiple versions for individual components) and extending the runtime in the future. Our goal is not to create a commercial quality XR product for current hardware. Instead, the goal for ILLIXR is to advance computer architecture, systems, and hardware-software co-design research for XR by making available a full system and key state-of-the-art components of both modern and future XR applications.
+1.  [Scene reconstruction][2] **
 
-Many of the current components of ILLIXR were developed by domain experts and obtained from publicly available repositories. They were modified for one or more of the following reasons: fixing compilation, adding features, or removing extraneous code or dependencies. Each component not developed by us is available as a forked github repository for proper attribution to its authors.
+1.  [Eye tracking][3] **
+
+1.  [Ambisonic encoding][4]
+
+1.  [Ambisonic manipulation and binauralization][4]
+
+1.  [Lens distortion correction][5]
+
+1.  [Chromatic aberration correction][5]
+
+1.  [Time warp][5]
+
+1.  [Computational holography for adaptive multi-focal displays][6] **
```

Comment:
weird numbering again, also I am curious what is the rationale behind the ordering?

Reply:
No rationale (kept the existing order). Could change it to alphabetical or something else as well.

Reply:
Thanks for pointing out the order. For simplicity, we should globally use the following order (similar to what we decided for the paper): first perception components, then visual, then audio. Alphabetical within each pipeline.